templates/index.html

```diff
@@ -56,55 +50,91 @@
 
                 <div class="row">
                     <div class="col-sm-12 form-group">
-                        <label for="Output" class='tr' key='res' size="+3"> RESULT:</label>
+                        <label for="Output" class='tr' id='res' size="+3"> RESULT:</label>
                         <textarea class="form-control tr" key='reso' type="textarea" size="+3" name="Output" id="take"
                             placeholder="" maxlength="6000" rows="5"></textarea>
                     </div>
                 </div>
 
                 <input type="text" id='takes'>
 
-                <div id=":rg" class="Am Al editable LW-avf tS-tW" hidefocus="true" aria-label="Message Body"
-                    g_editable="true" role="textbox" aria-multiline="true" contenteditable="true" tabindex="1"
-                    style="direction: ltr; min-height: 269px;" itacorner="6,7:1,1,0,0" spellcheck="false">
-                </div>
+
+
+                <form method="POST">
+                    <select id="lang_select">
+                        <option id='none' value="Select language">Select Language</option>
+                        <option id='lang1' value="irish">Irish</option>
+                        <option id='lang2' value="english">English</option>
+                    </select>
+                </form>
+
 
             </div>
         </div>
     </div>
 
+</body>
 
-    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
 
-    <script type="text/javascript">
+<script type="text/javascript" src="https://code.jquery.com/jquery-1.7.1.min.js"></script>
+<script src="https://code.jquery.com/ui/1.13.0/jquery-ui.js"></script>
+<script>
+    function Irish() {
+        document.getElementById("sent").innerHTML = "ENTER DO SENTENCE:";
+        document.getElementById("res").innerHTML = "TAIGHDE:";
+        document.getElementById("head").innerHTML = "SEICEÁIL SPELL";
+    }
+    function English() {
+        document.getElementById("sent").innerHTML = "ENTER YOUR SENTENCE:";
+        document.getElementById("res").innerHTML = "RESULT:";
+        document.getElementById("head").innerHTML = "SPELL CHECKER";
+    }
+</script>
+
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+
+<script type="text/javascript">
+
+    $(document).ready(function () {
+        var temp = 'english'
+        $('#lang_select').change(function () {
+            temp = $("option:selected").val();
+            /*$.ajax({
```

Comment:
How about we delete this code if we are not using this

Reply:
sure, will do it

Reply:
yes, why don't you delete the dead code ? by deleting the dead code  you can reduce the code and simplify.